espnet2/asr/pit_espnet_model.py

```diff
@@ -91,7 +91,7 @@ def pair_loss(permutation):
         )  # (batch_size, num_perm)
 
         min_losses, min_ids = torch.min(losses, dim=1)
-        opt_perm = all_permutations[min_ids]  # (batch_size, num_ref)
+        opt_perm = all_permutations[min_ids.cpu()]  # (batch_size, num_ref)
```

Comment:
why did you use CPU here?

Reply:
It's a bug fix. Maybe it shouldn't be in this PR. I found that `min_ids` is cuda tensor, and couldn't be used as the indices in slicing a tensor.

Reply:
Can you leave the comments in the source code then?